tests/integration/test_repeat.py

```diff
@@ -133,9 +175,14 @@ def test_array_2d_repeats_invalid(repeats):
 @pytest.mark.parametrize("repeats", (-3, [-3]))
 def test_array_1d_repeats_fatal_error(arr, repeats):
     anp = np.array(arr)
-    # numpy raises "ValueError: negative dimensions are not allowed"
-    # while cunumeric got "Fatal Python error: Aborted"
-    num.repeat(anp, repeats)
+    anum = num.array(arr)
+    expected_exc = ValueError
+    with pytest.raises(expected_exc):
+        np.repeat(anp, repeats)
+        # numpy raises "ValueError: negative dimensions are not allowed"
+    with pytest.raises(expected_exc):
+        num.repeat(anum, repeats)
+        # cuNumeric got "Fatal Python error: Aborted"
```

Comment:
Has it been confirmed that this is still happening? Good time to check, if not

Reply:
This error still exists